test/mockoptimizer.jl

```diff
@@ -58,25 +58,46 @@ end
     optimizer = MOIU.MockOptimizer(ModelForMock{Float64}())
     config = MOIT.TestConfig()
     optimizer.evalobjective = true
-    optimizer.optimize! = (optimizer::MOIU.MockOptimizer) -> mock_optimize!(optimizer, [1.0, 0.0, 2.0], (MOI.VectorOfVariables,             MOI.Nonnegatives) => [[0, 2, 0]],
-                                                                                                        (MOI.VectorAffineFunction{Float64}, MOI.Zeros)        => [[-3, -1]])
-    MOIT.lin1vtest(optimizer, config)
-    optimizer.optimize! = (optimizer::MOIU.MockOptimizer) -> mock_optimize!(optimizer, [1.0, 0.0, 2.0], (MOI.VectorAffineFunction{Float64}, MOI.Nonnegatives) => [[0, 2, 0]],
-                                                                                                        (MOI.VectorAffineFunction{Float64}, MOI.Zeros)        => [[-3, -1]])
-    MOIT.lin1ftest(optimizer, config)
-    optimizer.optimize! = (optimizer::MOIU.MockOptimizer) -> mock_optimize!(optimizer, [-4, -3, 16, 0], (MOI.VectorOfVariables,             MOI.Nonnegatives) => [[0]],
-                                                                                                        (MOI.VectorOfVariables,             MOI.Nonpositives) => [[0]],
-                                                                                                        (MOI.VectorOfVariables,             MOI.Zeros)        => [[7]],
-                                                                                                        (MOI.VectorAffineFunction{Float64}, MOI.Zeros)        => [[7, 2, -4]])
-    MOIT.lin2vtest(optimizer, config)
-    optimizer.optimize! = (optimizer::MOIU.MockOptimizer) -> mock_optimize!(optimizer, [-4, -3, 16, 0], (MOI.VectorAffineFunction{Float64}, MOI.Nonnegatives) => [[0]],
-                                                                                                        (MOI.VectorAffineFunction{Float64}, MOI.Nonpositives) => [[0]],
-                                                                                                        (MOI.VectorAffineFunction{Float64}, MOI.Zeros)        => [[7, 2, -4], [7]])
-    MOIT.lin2ftest(optimizer, config)
-    optimizer.optimize! = (optimizer::MOIU.MockOptimizer) -> mock_optimize!(optimizer)
-    MOIT.lin3test(optimizer, config)
-    optimizer.optimize! = (optimizer::MOIU.MockOptimizer) -> mock_optimize!(optimizer)
-    MOIT.lin4test(optimizer, config)
+
+    @testset "Conic linear tests" begin
+        optimizer.optimize! = (optimizer::MOIU.MockOptimizer) -> mock_optimize!(optimizer, [1.0, 0.0, 2.0], (MOI.VectorOfVariables,             MOI.Nonnegatives) => [[0, 2, 0]],
```

Comment:
This spacing is weird

Reply:
This is to align set types

Reply:
![image](https://user-images.githubusercontent.com/1733683/36474965-8c77d0d6-16f9-11e8-9808-cf5a39e6fb5f.png)
It would be better to have the lines fit within GitHub's column limit. They're aligned but still don't display very well.

Reply:
It should be ok now :)